recipes/tinyobjloader/all/conandata.yml

```diff
@@ -1,4 +1,26 @@
 sources:
+  "2.0.0-rc10":
```

Comment:
🤔 we typically never add pre prereleases, is this one special?

Reply:
@prince-chrismc the upstream have been working on RC since 2019. They will not release the final version soon 😞